lxd/network/driver_bridge.go

```diff
@@ -212,71 +268,368 @@ func (n *bridge) Validate(config map[string]string) error {
 
 			return nil
 		}),
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=bridge.hwaddr)
+		//
+		// ---
+		//  type: string
+		//  shortdesc: MAC address for the bridge
 		"bridge.hwaddr": validate.Optional(validate.IsNetworkMAC),
-		"bridge.mtu":    validate.Optional(validate.IsNetworkMTU),
-		"bridge.mode":   validate.Optional(validate.IsOneOf("standard", "fan")),
-
+		// lxdmeta:generate(entities=network-bridge; group=network-conf; key=bridge.mtu)
+		// The default value varies depending on whether the bridge uses a tunnel or a fan setup.
```

Comment:
What are the different default values? They should be specified in defaultdesc.

Reply:
@ru-fu If the the bridge is in `fan` mode, the max MTU is `1480` if `fan.type=ipip` and the max MTU is `1450` if `fan.type=vxlan`

Reply:
@ru-fu what do think about:

```
// lxdmeta:generate(entities=network-bridge; group=network-conf; key=bridge.mtu)
// The default value varies depending on whether the bridge uses a tunnel or a fan setup.
// ---
//  type: integer
//  defaultdesc: `1500` (`1480` for `fan.type=ipip`, `1450` for `fan.type=vxlan`)
//  shortdesc: Bridge MTU
```

Reply:
I think we should specify all three options on the same level, thus:

```
defaultdesc: `1500` if `bridge.mode=standard`, `1480` if `bridge.mode=fan` and `fan.type=ipip`, or `1450` if `bridge.mode=fan` and `fan.type=vxlan`
```

Reply:
fixed